site/generate.site.mjs

```diff
@@ -0,0 +1,36 @@
+import showdown from 'showdown';
+import { JSDOM, VirtualConsole } from 'jsdom';
+import * as fs from 'fs';
+
+const readme = fs.readFileSync('README.md', 'utf-8');
```

Comment:
I was unaware that there was even a option for `utf-8` on `readFileSync` I dont think you need it (I never see it used?)

Reply:
without specifying the encoding, I got a `<Buffer 3c 21 44[...]` that's why I added it.